src/main/java/core/basesyntax/service/serviceimpl/FruitFileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.exception.NoFileToReadException;
+import core.basesyntax.service.interfaces.FruitFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FruitFileReaderImpl implements FruitFileReader {
```

Comment:
Again, there is not logic that works only with fruits, it is universal and reusable with any file, so no need to specify that
```suggestion
public class FileReaderImpl implements FileReader {
```